test/definitions/test_helper.rb

```diff
@@ -84,6 +84,9 @@ def refute_scenario(filter)
 end
 
 TEST_DIR = File.dirname(__FILE__)
-ForemanMaintain.setup(
-  :config_file => File.join(TEST_DIR, 'config/foreman_maintain.yml.test')
-)
+CONFIG_FILE = File.join(TEST_DIR, 'config/foreman_maintain.yml.test').freeze
+STORAGE_DATA_FILE = '/lib/foreman-maintain/data_test.yml'.freeze
```

Comment:
That's strange. `require` on a constants file for example `foreman_maintain/constants.rb` should ideally work. But the tests are failing, looks like it's not able to recognize definitions under test folder.
Few of the errors are:
```
NameError: uninitialized constant Scenarios::PresentUpgrade
NameError: uninitialized constant Procedures::Setup
NameError: uninitialized constant Checks::PresentServiceIsRunning
```
And failures:
```
 18) Failure:
ForemanMaintain::Cli::HealthCommand::list-tags#test_0001_lists the defined tags [/root/foreman_maintain/test/lib/cli/health_command_test.rb:50]:
--- expected
+++ actual
@@ -1,3 +1,4 @@
 "[default]
-[pre-upgrade-check]
+[post-upgrade]
+[pre-upgrade]
```